classes/Forms.js

```diff
@@ -18,12 +18,19 @@ module.exports = class Forms extends OneBlinkAPI {
   async generateFormUrl(
     formId /* : ?mixed */,
     externalId /* : ?mixed */,
-    preFillData /* : ?mixed */
+    preFillData /* : ?mixed */,
+    expiryInSeconds /* : ?mixed */
```

Comment:
Can this please be changed to an object now and added to the changelog as a BREAKING CHANGE. Its not very nice to use a function like so:

```js
generateFormUrl(1, undefined, undefined, 600)
```

This would be much better


```js
generateFormUrl({
  formId: 1,
  expiryInSeconds: 600
})
```

Reply:
Changelog should have something like this:

### Breaking Change

- `Forms.generateFormUrl()` now takes and object with the following properties
  - `formId`
  - `externalId`
  - `preFillData`
  - `expiryInSeconds`


```diff
const OneBlink = require('@oneblink/sdk')

const options = {
  accessKey: '123455678901ABCDEFGHIJKL',
  secretKey: '123455678901ABCDEFGHIJKL123455678901ABCDEFGHIJKL'
}
const formsSDK = new OneBlink.Forms(options)

-const formId = 1
-const externalId = 'My Custom Identifier'
-const preFilledData = {
-  'FieldName1' : 'A Machine',
-  'FieldName2' : 'Room B'
-}
-formsSDK.generateFormUrl(formId, externalId, preFilledData, expiryInSeconds)
+formsSDK.generateFormUrl({
+  formId: 1,
+  externalId: 'My Custom Identifier',
+  preFilledData: {
+    'FieldName1' : 'A Machine',
+    'FieldName2' : 'Room B'
+  }
+})
```